index.js

```diff
@@ -176,18 +176,18 @@ function getExistingPath (command, opts) {
 
 module.exports._getNpmCache = getNpmCache
 function getNpmCache (opts) {
-  return which(opts.npm).then(npmPath => {
-    const args = ['config', 'get', 'cache', '--parseable']
+  return which('node').then(nodePath => {
+    const args = [opts.npm, 'config', 'get', 'cache', '--parseable']
```

Comment:
This would left the current implementation untouched and just add a fallback for windows and bundled npm (and .js files in --npm switch).